### PR TITLE
Update experimental test code qos_policy_config_test.go

### PR DESCRIPTION
**Pull request by sezhang2**
Added the config verification code. It needs to be enabled by resoling TODO items when feature is supported.

### CODE REVIEW

File: feature/experimental/qos/ate_tests/qos_policy_config_test/qos_policy_config_test.go [hunk 1]
```diff
@@ -204,15 +207,56 @@ func TestQoSClassifierConfig(t *testing.T) {
...
-	// t.Logf("qosClassifiers from telmetry: %v", qosClassifiers)
+		// TODO: Remove the following t.Skipf() after the config verification code has been tested.
+		t.Skipf("Skip the QoS config verification until it is tested against a DUT.")
+
+		classifier := gnmi.OC().Qos().Classifier(tc.name)
+		term := classifier.Term(tc.termID)
+		action := term.Actions()
+		condition := term.Conditions()
+
+		cmp.Equal([]uint8{1, 2, 3}, []uint8{1, 2, 3})
```
Comment (nraghavendran) [inline feature/experimental/qos/ate_tests/qos_policy_config_test/qos_policy_config_test.go:227]: a comment on what this does will be helpful

File: feature/experimental/qos/ate_tests/qos_policy_config_test/qos_policy_config_test.go [hunk 2]
```diff
@@ -204,15 +207,56 @@ func TestQoSClassifierConfig(t *testing.T) {
...
+		cmp.Equal([]uint8{1, 2, 3}, []uint8{1, 3, 2})
+
+		if got, want := gnmi.Get(t, dut, classifier.Name().State()), tc.name; got != want {
```
Comment (nraghavendran) [inline feature/experimental/qos/ate_tests/qos_policy_config_test/qos_policy_config_test.go:230]: consider changing "got,want"  to something like "result_obtained,result_expected"

File: feature/experimental/qos/ate_tests/qos_policy_config_test/qos_policy_config_test.go (hunk 1 above)
Comment (sezhang2) [reply feature/experimental/qos/ate_tests/qos_policy_config_test/qos_policy_config_test.go:228]: done

File: feature/experimental/qos/ate_tests/qos_policy_config_test/qos_policy_config_test.go (hunk 2 above)
Comment (sezhang2) [reply feature/experimental/qos/ate_tests/qos_policy_config_test/qos_policy_config_test.go:231]: got, want are recommended terms for go readability. The log indicated what values the test code got and want.

**Pull request closed without merging**